frontend/src/hooks/useFormData/index.tsx

```diff
@@ -172,10 +196,23 @@ const useFormData = (): [FormData, Meta, Helpers] => {
     useControlledClearEffect(formData.didFellowshipReachMordor.value, "mordor");
     useControlledClearEffect(formData.wasAragornCrowned.value, "aragornTurn");
 
+    strongholds.filter(isStrongholdConditional).map((stronghold) => {
+        useStrongholdDeselectEffect(
+            stronghold,
+            isStrongholdInPlay(formData.expansions.value, stronghold)
+        );
+    });
```

Comment:
Mentioned having issues with this bit in Discord, but then I added a filter for `isStrongholdConditional` so it will only generate the three `useEffect`s actually needed. I can live with generating them dynamically - `isStrongholdConditional` will never return a different result, so there's no danger of a runtime error, and I just was having an irrational feeling. TLDR; scratch everything I said in Discord